### ISSUE TITLE
Can't get torrentes

### ISSUE THREAD

**Issue by Zexuz** [Zexuz/TvCalander#26]
```
Sending 'PUT' request to URL : http://localhost:8080/api/ImdbService/v1/Series/0364845
Response Code : 200
/recent/0
```

``````
java.lang.NullPointerException
        at com.webcrawler.site.sites.TPB.getTorrentsForSite(TPB.java:78)
        at com.webcrawler.managers.SiteManager.getRecentTorrentPages(SiteManager.java:40)
        at com.webcrawler.managers.Managers.tick(Managers.java:95)
        at com.webcrawler.MainThread.run(MainThread.java:26)
        at java.lang.Thread.run(Thread.java:745)
Exception in thread "Main thread" java.lang.NullPointerException
        at java.util.ArrayList.addAll(ArrayList.java:577)
        at com.webcrawler.managers.SiteManager.getRecentTorrentPages(SiteManager.java:40)
        at com.webcrawler.managers.Managers.tick(Managers.java:95)
        at com.webcrawler.MainThread.run(MainThread.java:26)
        at java.lang.Thread.run(Thread.java:745)
kalle@kalle-HP-EliteBook-8560p:~/TvCalander$```
``````

**Comment by Zexuz** [comment]
It looks like this is due to TPB getting a mobile friendly site.

**Comment by Zexuz** [comment]
Fixed in #27
